lego/apps/events/models.py

```diff
@@ -692,22 +692,18 @@ def waiting_registrations(self):
     def waiting_registration_count(self):
         return self.waiting_registrations.count()
 
-    @property
-    def is_abakom_only(self):
-        return (
-            self.require_auth
-            and self.can_view_groups.count() == 1
-            and self.can_view_groups.filter(name="Abakom").exists()
-        )
-
-    def set_abakom_only(self, abakom_only):
-        abakom_group = AbakusGroup.objects.get(name="Abakom")
-        if abakom_only:
+    def set_groups_with_view_permission(self, groups_with_view_permission):
+        if len(groups_with_view_permission) > 0:
+            self.groups_with_view_permission.clear()
             self.require_auth = True
-            self.can_view_groups.add(abakom_group)
+            for group in groups_with_view_permission:
+                abakus_group = AbakusGroup.objects.get(name=group)
+                self.groups_with_view_permission.add(abakus_group)
+                self.can_view_groups.add(abakus_group)
```

Comment:
These two fields seem to be very similar. What is the difference? Do you need the `groups_with_view_permission` field?